Source/interactivity_manager_impl.cpp

```diff
@@ -1229,18 +1246,23 @@ void interactivity_manager_impl::process_reply(const web::json::value& jsonReply
                 else
                 {
                     LOGS_INFO << L"Unhandled reply: " << jsonReply.serialize().c_str() << "to message " << message->to_string();
+                    messageWasProcessed = false;
                 }
             }
         }
         else
         {
             LOGS_INFO << "Unexpected json reply, no id";

```

Comment:
>LOGS_INFO << "Unexpected json reply, no id"; [](start = 12, length = 44)

Again, this should be a true error. A message with no message id is not a valid message or response (unless the RPC protocol has dramatically changed over the last few months).